src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -114,54 +69,60 @@ export interface Cohort extends DatasetBuilderType {
   criteriaGroups: CriteriaGroup[];
 }
 
-export interface DatasetBuilderType {
-  name: string;
-}
-
 export type DatasetBuilderValue = DatasetBuilderType;
 
 export type ValueSet = {
   domain: string;
   values: DatasetBuilderValue[];
 };
 
-export interface GetConceptsResponse {
-  result: Concept[];
-}
-
-export interface SnapshotBuilderParentConcept {
-  parentId: number;
-  children: Concept[];
-}
-
-export interface GetConceptHierarchyResponse {
-  readonly result: SnapshotBuilderParentConcept[];
-}
-
-export interface SearchConceptsResponse {
-  result: Concept[];
-}
-
 export type DatasetRequest = {
   cohorts: Cohort[];
-  conceptSets: ConceptSet[];
+  conceptSets: SnapshotBuilderDatasetConceptSet[];
   valueSets: ValueSet[];
 };
 
-export type DatasetAccessRequest = {
+export type SnapshotAccessRequest = {
   name: string;
   researchPurposeStatement: string;
   datasetRequest: DatasetRequest;
 };
 
-export const convertValueSet = (valueSet: ValueSet): ValueSetApi => {
+export type SnapshotBuilderCountRequest = {
+  cohorts: Cohort[];
+};
+
+export interface DomainOption extends SnapshotBuilderOption {
+  kind: 'domain';
+  root: SnapshotBuilderConcept;
+  conceptCount?: number;
+  participantCount?: number;
+}
+
+export type BuilderSettings = {
```

Comment:
It looks like this type and `DatasetModel` aren't being used anywhere except in test code. Can you show me where this is used in the UI code? If they're not used, I think the fix here is to replace the test code usages with the API types.

Reply:
I removed `DatasetModel`. The only reason I made `DatasetModel` was because `SnapshotBuilderDomainOption` removed `conceptCount` and `participantCount`.